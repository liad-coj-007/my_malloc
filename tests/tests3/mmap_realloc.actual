                               Memory Statistics
                               =================
Free blocks: 32
Free bytes: 4323060
Allocated blocks: 33
Allocated bytes: 4324352
Meta data bytes: 1056
                               Memory Statistics
                               =================
Free blocks: 32
Free bytes: 4193280
Allocated blocks: 32
Allocated bytes: 4193280
Meta data bytes: 1024
