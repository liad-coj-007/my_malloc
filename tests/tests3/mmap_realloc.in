add 1 32768
realloc 1 323 2
print_stats
free 2
print_stats