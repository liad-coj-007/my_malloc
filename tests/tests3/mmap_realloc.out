                               Memory Statistics
                               =================
Free blocks: ME
Free bytes: MH + MV - 323*4
Allocated blocks: ME+1
Allocated bytes: MH+MV 
Meta data bytes: ME*ME+ME
                               Memory Statistics
                               =================
Free blocks: ME
Free bytes: MH
Allocated blocks: ME
Allocated bytes: MH 
Meta data bytes: ME*ME